projects/storefrontapp-e2e-cypress/cypress/helpers/coupons/cart-coupon.ts

```diff
@@ -129,6 +136,15 @@ function registerCartRefreshRoute() {
   cy.intercept('GET', exp1).as('refresh_cart');
```

Comment:
may you please change it to:
`cy.intercept({
      method: 'GET',
      pathname: `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
    'BASE_SITE'
  )}/products/${productCode}?fields=*&lang=en&curr=USD`,
    }).as('xxx');`
    
    and replcae exp1 with some meaningful name